src/components/Dialog.jsx

```diff
@@ -39,9 +38,7 @@ class Dialog extends React.Component {
         <div
           className="popup dialog"
           tabIndex="0"
-          ref={(c) => { this.popupBody = c; }}
           role="button"
-          onClick={(e) => { return e.target === this.popupBody && this.close(e); }}
```

Comment:
Reminder: this code was removed because the <Dialog> component doesn't have screen-spanning div that blocks interactions with elements behind it. This used to exist for a separate <Popup> _modal_ component, used in ASM.